content/geographical-unit-data.md

```diff
@@ -128,7 +128,105 @@ library(PBSmapping)
 shapefile <- importShapefile("/my_shapefile.shp")
 ```
 
-## Weighted aggregations within spatial units
+## Weighted averages within spatial units
+
+Now, you probably have a gridded spatiotemporal dataset of historical
+weather and economic output data specific to shapefile regions.  The
+next step is construct the weather measures that will correspond to
+each of your economic data observations.  To do this, you will need to
+construct a weighted average of the weather in each region for each
+timestep.
+
+This can be a computationally expensive operation, so it is important
+to set up the mathematical process efficiently. The regional averaging
+process is equivalent to a matrix transformation:
+$$w_\text{region} = A w_\text{gridded}$$
+
+where $$w_\text{region}$$ is a vector of weather values across
+regions, in a given timestep; and $$w_\text{gridded}$$ is a vector of
+weather values across grid cells.  Suppose there are $$N$$ regions and
+$$R C$$ grid cells, then the transformation matrix $$A$$ will be $$N x
```

Comment:
These ones need to be one $ symbol (ie. $w_\text{region}$ instead of $$w_\text{region}$$), otherwise it will put the formulas in new lines.

Reply:
Done.